Dockerfile

```diff
@@ -1,22 +1,22 @@
 ### Base Image
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.6 as base
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.0.0 as base
 
 WORKDIR /opt/cloudigrade
 
-RUN rpm -iv https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
-    && microdnf update \
+RUN rpm -iv https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
+    && microdnf update -y \
     && microdnf install -y \
         git \
         jq \
         libicu \
         nmap-ncat \
         postgresql14-libs \
         procps-ng \
-        python39 \
+        python3 \
```

Comment:
Unfortunately, the dnf repos no longer use the more precise name `python39` and instead just call it `python3`. ☹️ 

```
❯ docker run --privileged -it --entrypoint bash registry.access.redhat.com/ubi9/ubi-minimal:9.0.0
[root@a5ae1b0c68b5 /]# microdnf -y update > /dev/null 2>&1 && microdnf -y install dnf > /dev/null 2>&1
[root@a5ae1b0c68b5 /]# dnf search python39
Red Hat Universal Base Image 9 (RPMs) - BaseOS                                                                                           1.4 MB/s | 542 kB     00:00
Red Hat Universal Base Image 9 (RPMs) - AppStream                                                                                        3.2 MB/s | 1.4 MB     00:00
Red Hat Universal Base Image 9 (RPMs) - CodeReady Builder                                                                                 41 kB/s |  12 kB     00:00
No matches found.
[root@a5ae1b0c68b5 /]# dnf search python3
Last metadata expiration check: 0:00:01 ago on Mon Jul 25 19:54:11 2022.
===================================================================== Name Exactly Matched: python3 =====================================================================
python3.x86_64 : Python 3.9 interpreter
...
```

Reply:
no worries,  what does `python --version` show ?

Reply:
```
❯ docker run --privileged -it --entrypoint bash registry.access.redhat.com/ubi9/ubi-minimal:9.0.0
[root@63c43753ee96 /]# microdnf install -y python3 > /dev/null 2>&1
[root@63c43753ee96 /]# python --version
Python 3.9.10
```
(for now)

Reply:
That's fine, at least we can update our local dev python to match exactly and somehow keep an eye on that puppy periodically.

Reply:
FWIW, I originally discovered this because I went searching for a Python _3.10_ package out of curiosity, but one does not yet appear to be available for ubi9/ubi-minimal.